src/main/java/graphql/schema/idl/SchemaPrinter.java

```diff
@@ -458,7 +511,7 @@ private static String printAst(Object value, GraphQLInputType type) {
 
             // when serializing a GraphQL schema using the type system language, a
             // schema definition should be omitted if only uses the default root type names.
-            boolean needsSchemaPrinted = options.includeSchemaDefinition;
+            boolean needsSchemaPrinted = options.isIncludeSchemaDefinition();
```

Comment:
Any reason for adding the `is` at the start, seems to flow easier without it.

Reply:
one was a field - this is now the public method.  Not sure why I changed it.  Consistency elsewhere??  Not sure